.github/workflows/lint-pipelines.yml

```diff
@@ -9,29 +9,57 @@ on:
   pull_request:
 
 jobs:
-  lint:
-    name: Lint GoCD Pipelines
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+    render:
+        name: Render GoCD Pipelines with Jsonnet
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c  # v3
+          - uses: getsentry/action-gocd-jsonnet@v0
+            with:
+              jb-install: true
+              check-for-changes: true
+              convert-to-yaml: true
+              jsonnet-dir: gocd/templates
+              generated-dir: gocd/generated-pipelines
 
-      - uses: getsentry/action-setup-gocd-cli@8391790b37dcf61bdf52d13fa1bf044c9bee9ac9 # v1.0.0
+    files-changed:
+        name: files-changed
+        runs-on: ubuntu-latest
+        # Map a step output to a job output
+        outputs:
+            gocd: ${{ steps.changes.outputs.gocd }}
+        steps:
+          - name: Check for relevant file changes
+            uses: getsentry/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+            id: changes
+            with:
+              filters: |
+                gocd:
+                  - 'gocd/**'
 
-      - name: Lint Pipelines with gocd-cli
-        run: |
-          echo "GoCD YAML Linting"
-          find "gocd" -name "*.yaml" -type f -print0 | \
-          xargs -0 -I'{}' bash -c 'printf  "\n🔎 Linting {}\n\t" && gocd-cli configrepo syntax --yaml --raw "{}"'
+    validate:
+        if: needs.files-changed.outputs.gocd == 'true'
+        needs: files-changed
+        name: Validate GoCD Pipelines
+        runs-on: ubuntu-latest
 
-  render:
```

Comment:
I'd still like to keep this as it ensures the jsonnet templates render

Reply:
I moved it, wasn't deleted.